README.md

```diff
@@ -316,6 +318,61 @@ class PresentationViewController: UIViewController {
 
 You can learn more about `EpoxyPresentations` in its [wiki entry](https://github.com/airbnb/epoxy-ios/wiki/EpoxyPresentations).
 
+## EpoxyLayoutGroups
+
+LayoutGroups are UIKit [Auto Layout](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/AutolayoutPG/index.html) containers inspired by SwiftUI's [`HStack`](https://developer.apple.com/documentation/swiftui/hstack) and [`VStack`](https://developer.apple.com/documentation/swiftui/vstack) that allow you to easily compose UIKit elements into horizontal and vertical groups.
+
+`VGroup` allows you to group components together vertically to create stacked components like this:
+
+| ActionRow |
+| --------- |
+| ![ActionRow screenshot](docs/images/ActionRow.png) |
+
+```swift
+// create any number of views you want to vertically group
+let titleLabel = UILabel(...)
+let subtitleLabel = UILabel(...)
+let actionButton = UIButton(...)
+
+// add them to the group
+let group = VGroup(alignment: .leading, spacing: 8) {
+  titleLabel
+  subtitleLabel
+  actionButton
+}
```

Comment:
Don't we want to be advertising the GroupItem syntax? As is this looks like the builder is taking constrainables